deployments/ticdc/docker/integration-test.Dockerfile

```diff
@@ -55,5 +55,5 @@ COPY . .
 # We always need to clean before we build, please don't adjust its order.
 RUN make clean
 RUN make integration_test_build kafka_consumer cdc
-COPY --from=downloader /root/download/bin/* ./bin
+COPY --from=downloader /root/download/bin/* ./bin/
```

Comment:
Otherwise, it may report an error in some dockers.

Reply:
I haven't encountered it on my own computer, but @3AceShowHand  seems to have encountered the problem.